index.html

```diff
@@ -16,7 +16,7 @@
     <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins:wght@300&family=Roboto:wght@100;400&display=swap" rel="stylesheet">
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
     <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@300&family=Poppins:wght@300&family=Roboto:wght@100;400&display=swap" rel="stylesheet">
-    <title>Carlyle's-Portfolio-site</title>
+    <title>Chia Carlyle's Portfolio Site</title>
```

Comment:
- [ ] If you click on line 19 or 20, or any other line, you can see the URL changing. For example, by clicking line 19 we can see the URL https://github.com/carlylechia/MY-PORTFOLIO/pull/8/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R19. You can copy this link for your report, for your the first point:

- Page titles - one issue found [link](https://github.com/eng-mohamed-eldeeb/protofolio/pull/5/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051L22)

Please, consider this technic for all found issues.